client/signup/jetpack-connect/index.jsx

```diff
@@ -17,7 +17,7 @@ import Main from 'components/main';
 import JetpackConnectNotices from './jetpack-connect-notices';
 import SiteURLInput from './site-url-input';
 import { getSiteByUrl } from 'state/sites/selectors';
-import { requestSites } from 'state/sites/actions';
+import QuerySites from 'components/data/query-sites';
```

Comment:
as we talked in slack, it seems that #6477 is needed for using computed attributes in sites, and without it, there can be problems related with mapped domains.

Since we are already using redux for sites, and we already are affected by this potential problems, I think we can consider this PR mergeable before #6477 is ready